pkg/controller/llmisvc/workload_multi_node.go

```diff
@@ -531,11 +533,32 @@ func (r *LLMInferenceServiceReconciler) propagateLeaderWorkerSetMetadata(llmSvc
 			}
 			expected.Annotations[k] = v
 		}
+
+		if strings.HasPrefix(k, "k8s.v1.cni.cncf.io") {
```

Comment:
maybe we can just  `|| strings.HasPrefix(k, "k8s.v1.cni.cncf.io")` in the first `if`?